aea/helpers/profiling.py

```diff
@@ -86,56 +88,50 @@ class Profiling(Runnable):
 
     def __init__(
         self,
+        types_to_track: List[Type],
         period: int = 0,
-        objects_instances_to_count: List[Type] = None,
-        objects_created_to_count: List[Type] = None,
```

Comment:
Why are we no longer keeping track of the created count?

Reply:
We are, but if felt like specifying two lists of types was unnecesary. What I've proposed now is to have just one list (types_to_track) and keep track of the calls to __new__ and also __del__ in self.object_counts which stores the info for both.